batch-1/movie-info/akshay/filminfo.rb

```diff
@@ -0,0 +1,88 @@
+require 'open-uri'
+require 'json'
+
+
+
+def input
+	puts"Enter the Movie title:"
+	movie_name=gets.chomp
+	movie_name=movie_name.split(" ").join("+")
+	return movie_name
+end
+
+def generate_page movie_name
+	movie_url= "http://www.omdbapi.com/?t=#{movie_name}&y=&plot=short&r=json"
+	content = open(movie_url).read
+	content_hash=JSON.parse(content)
+	html_file=File.new("movie_detail.html","w")
+	html_file.puts("<!DOCTYPE html>")
```

Comment:
Instead of writing HTML inside ruby code, how about this approach:

2 files: filminfo.rb and template.html

In ```template.html```, we would write something like this:

```
....
<body>
  <h1>{{title}}</h1>
</body>
...
```

We would open ```template.html``` as ```File``` in ruby and replace ```{{title}}``` with appropriate text using string replace methods. That way we could take the entire html out of ruby code and keep it in a separate template file. Give that a try.